src/sql/engine/px/ob_px_task_process.cpp

```diff
@@ -439,6 +439,10 @@ int ObPxTaskProcess::do_process()
     if (OB_SUCC(ret)) {
       if (nullptr != arg_.op_spec_root_) {
         const ObPxSqcMeta &sqc_meta = arg_.sqc_handler_->get_sqc_init_arg().sqc_;
+        const ObQcRecordData & record_data = sqc_meta.get_record_data();
+        // 展示从qc发来的监控数据
+        LOG_DEBUG("receive information from qc, query", K(record_data.sql), 
```

Comment:
U can use the ```K(record_data )``` by implementing the ```TO_STRING_KV``` function.

Reply:
> U can use the `K(record_data )` by implementing the `TO_STRING_KV` function.

done